plugins/net_plugin/net_plugin.cpp

```diff
@@ -3758,94 +3773,24 @@ namespace eosio {
                   ("bt", best_head)("cid", cid)("num", block_num)("id", obh->id()) );
          my_impl->dispatcher.add_peer_block( obh->id(), cid ); // no need to send back to sender
          my_impl->dispatcher.bcast_block( obh->block(), obh->id() );
+         c->block_status_monitor_.accepted();
 
          if (best_head) {
-            fc_dlog(logger, "posting block ${n} to app thread", ("n", ptr->block_num()));
+            ++c->unique_blocks_rcvd_count;
```

Comment:
what is the meaning of `unique_blocks_rcvd_count`, i.e. what does it count?

Reply:
How many blocks this connection received first. Prometheus metric that will be removed soon.